sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/package-info.java

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+/**
+ * Package containing the implementations and inner classes for KeyVaultAccessControlClient. The key vault client
+ * performs cryptographic key operations and vault operations against the Key Vault service.
+ */
+package com.azure.security.keyvault.administration.implementation;
```

Comment:
There shouldn't be an "implementation" in there, should there?

Reply:
This was auto-generated. It looks ok to me.

Reply:
This file is for the implementation package, thus this is expected.